homeassistant/components/netgear_lte/__init__.py

```diff
@@ -23,10 +28,30 @@
 DOMAIN = 'netgear_lte'
 DATA_KEY = 'netgear_lte'
 
+
+NOTIFY_SCHEMA = vol.Schema({
+    vol.Optional(CONF_NAME, default=DOMAIN): cv.string,
+    vol.Optional(CONF_RECIPIENT, default=[]):
+        vol.All(cv.ensure_list, [cv.string]),
+})
+
+SENSOR_SCHEMA = vol.Schema({
+    vol.Required(CONF_MONITORED_CONDITIONS):
+        vol.All(cv.ensure_list, [vol.In(sensor_types.ALL_SENSORS)]),
+})
+
+DEFAULT_SENSOR = {
+    CONF_MONITORED_CONDITIONS: sensor_types.DEFAULT_SENSORS
+}
+
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: vol.All(cv.ensure_list, [vol.Schema({
         vol.Required(CONF_HOST): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(NOTIFY_DOMAIN, default={}):
+            vol.All(cv.ensure_list, [NOTIFY_SCHEMA]),
+        vol.Optional(SENSOR_DOMAIN, default=DEFAULT_SENSOR):
```

Comment:
Default to an empty dict here.

Reply:
If we default to the empty dict here it should get populated with the default items of the sensor schema.